### PR TITLE
#163088648 Deploy base application to Heroku

### PR DESCRIPTION
**Pull request by akhilome**
#### What does this PR do?

prepares the base app for heroku deployment

#### Description of Task to be completed?

- install and configure an express server
- setup babel to compile server code
- setup appropriate build scripts

#### How should this be manually tested?

- clone the repo and cd into it
- run `npm i` to install required project dependencies
- run `npm run build` to build a production-ready version of the react app
- run `npm start` to start the app
- visit `localhost:8080` in your browser to view the app

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker stories? (If applicable)

[#163088648](https://www.pivotaltracker.com/story/show/163088648)

#### Screenshots (If applicable)

image

#### Questions:

N/A

### CODE REVIEW

**Review by okoroemeka — APPROVED** [review]

**Comment by Johnsonojo** [comment]
Good Job @akhilome. But please fix the conflict.

File: app.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (frostyblok) [inline app.json:1]: Hello Kizito, what does the app.json file do?

Comment (oluwajuwon) [reply app.json:1]: @frostyblok the `app.json` file contains configurations for Heroku, that are needed to create review apps from the repository every time a new PR is raised.

Comment (akhilome) [reply app.json:1]: @frostyblok, good question.

The `app.json` is required by Heroku for the pipeline to create review apps based off of our staging branch

**Review by oluwajuwon — APPROVED** [review]
LGTM! 👍

**Review by Johnsonojo — APPROVED** [review]
Nice Job!

File: app.json (hunk 1 above)
Comment (frostyblok) [reply app.json:1]: Oh, I didn't know that, thank you! I will be implementing that on my CP

**Review by frostyblok — APPROVED** [review]
This is a good job, Kizito!

**Pull request merged**